lib/msal-node/test/test_kit/StringConstants.ts

```diff
@@ -175,6 +175,11 @@ export const TEST_CONFIG = {
     SID: "session-id",
     CORRELATION_ID: "7821e1d3-ad52-42t9-8666-399gea483401",
     CLAIMS: '{"access_token":{"example_claim":{"values":["example_value"]}}}',
+    NBF_CLAIMS:
```

Comment:
Adding a few more test data for additional coverage would be good, [this](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/a98bcf7fac02082673dbb7896236df6659818d49/tests/Microsoft.Identity.Test.Common/TestConstants.cs#L152) is what we have in MSAL .NET 

for e.g. 
```js
// Empty Claims JSON
const EmptyClaimsJson = {};

// Claims with an Additional Claim
const ClaimsWithAdditionalClaim = {
  access_token: {
    nbf: { essential: true, value: "1701477303" },
    additional_claim: { key: "value" },
  },
};

// Claim with Additional Key
const ClaimWithAdditionalKey = {
  ...ClaimsWithAdditionalClaim,
  new_claim: { new_key: "new_value" },
  some_other_key: {
    nbf: { essential: true, value: "1701477303" },
    additional_claim: { key: "value" },
    new_claim: { new_key: "new_value" },
  },
};

// Claim With Additional Key And AccessKey
const ClaimWithAdditionalKeyAndAccessKey = {
  some_other_key: {
    nbf: { essential: true, value: "1701477303" },
    additional_claim: { key: "value" },
    new_claim: { new_key: "new_value" },
  },
  access_token: {
    nbf: { essential: true, value: "1701477303" },
    additional_claim: { key: "value" },
    new_claim: { new_key: "new_value" },
  },
};
```